serialization.md

```diff
@@ -26,13 +26,6 @@ This specification uses the following terms:
 	Changing it means creating a new derived image, instead of changing the existing image.
     </dd>
     <dt>
-        Image Filesystem Changeset
-    </dt>
-    <dd>
-        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
-	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
```

Comment:
On Fri, Sep 02, 2016 at 03:15:54PM -0700, Stephen Day wrote:
> > @@ -26,13 +26,6 @@ This specification uses the following terms:
> >  	Changing it means creating a new derived image, instead of changing the existing image.
> >      </dd>
> >      <dt>
> > -        Image Filesystem Changeset
> > -    </dt>
> > -    <dd>
> > -        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
> > -	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
>
> I'd like to see most of this information integrated here. Repeating
> ideas reinforces them in the spec and provides context to link the
> reader to these important ideas.

“Here are some neat tricks for using layers efficiently” doesn't seem
like a core concept to me.  It sounds *useful* (hence the NOTE in
layer.md), but you can be a technically-correct layer author and
consumer without that information.  I'd rather restrict the
definitions here to properties that are fundamental to the term being
defined.

But it's only a line ;), so I've copied it over verbatim with b8ff53a
→ 2f6d7fd.